atomicfu-common/src/main/kotlin/kotlinx/atomicfu/AtomicFU.common.kt

```diff
@@ -73,6 +73,8 @@ public expect class AtomicRef<T> {
      */
     public var value: T
 
+    public val trace: Trace?
```

Comment:
Atomic classes shall NOT expose `trace` field. Please, remove. The correct usage pattern for trace is this:

```
val trace = Trace() // a separate field for trace
val a = atomic(0, trace)
```

Then later in code you can do `trace { "something" }`.

Bytecode transformer shall completely remove `trace` field from the source class, all its accessors, and references.